xml_inscriptions/transcribed/NY.Ith.CU.G.CCC_0620-mdn.xml

```diff
@@ -0,0 +1,190 @@
+                    </physDesc>
+
+                    <!--  *************************************  <history>  *********************************  -->
+
+                    <history>
+
+                        <summary>
+                            <rs>modern</rs>  <!-- content can be "modern" or "modern-text". A blank value indicates that the inscription is ancient.  -->
+                        </summary>
+
+                        <origin>
+                            <date evidence="#unspecified" notBefore="-0360" notAfter="-0360">c. 360
+                                BCE</date>
+                            <placeName ref="europe.greece.athens">Piraeus</placeName>
+                        </origin>
+                        <provenance type="observed" subtype="recorded">
+                            <p>Plaster cast was made between <date when="1890">1890</date> and <date when="1900">1900</date>.</p>
+                        </provenance>
+
+
+                    </history>
+                </msDesc>
+                <!--  *************************************  <listBibl>  *********************************  -->
+                <listBibl>
+                    <bibl>
+                        <ptr type="rest-of-bibl" target="#IG_II2"/>
+                        <biblScope unit="item">12562</biblScope>
+                    </bibl>
+                    <bibl>
+                        <ref type="external"
+                            target="https://digital.library.cornell.edu/catalog/ss:173248">Cornell web entry</ref>
+                    </bibl>
+                </listBibl>
+            </sourceDesc>
+        </fileDesc>
+
+        <!--  *************************************  <encodingDesc>  ********************************* -->
+
+        <encodingDesc>
+
+                <xi:include
+                    href="http://library.brown.edu/usep_data/resources/include_taxonomies.xml">
+                    <xi:fallback>
+                        <p>Taxonomies for US Epigraphy controlled values</p>
+                    </xi:fallback>
+                </xi:include>
+
+        </encodingDesc>
+
+        <profileDesc> <!-- not used --> </profileDesc>
+
+
+        <!--  *************************************  <revisionDesc>  ********************************* -->
+    	<!-- Record all changes here -->
+
+        <revisionDesc>
+            <change when="2025-02-22" who="Advaith Rege">Initial Encoding</change>
+        </revisionDesc>
+
+    </teiHeader>
+
+    <!--  *************************************  <facsimile>  *********************************  -->
+        <!-- Used to record images of an inscription.   -->
+
+    <facsimile>
+        <surface>
+            <desc/>
+            <graphic url="NY.Ith.CU.G.CCC_0620-mdn.jpg"/>
+        </surface>
+    </facsimile>
+
+    <text>
+        <body>
+            <div type="edition">
+            <ab>
+                <lb n="1"/>
+                <persName key="Pyrrias">Πυρρίας</persName>.
```

Comment:
Typically for keys for Greek names, put both the original and the transliteration, separated by a space; that way both get picked up in the indexing.